R/pixarfilms.R

```diff
@@ -44,7 +49,6 @@ dm_pixarfilms <- function(color = TRUE) {
   dm <-
     dm %>%
     dm_add_pk(pixar_films, film) %>%
-    dm_add_pk(pixar_people, c(film, role_type)) %>%
```

Comment:
Did you mean to remove this primary key?

Reply:
Yes, this is not unique and shouldn't be, you can have several screenwriters for a given movie for instance.